monitoring/local_monitoring/gpu-demo/no-gpu/kruize_gpu_demo.py

```diff
@@ -0,0 +1,493 @@
+import json
+import re
```

Comment:
Does this script setup kind? It failed for me at this step

```
 python3 kruize_gpu_demo.py 
Please check kruize_gpu_demo.log for complete log info



Checking Pre-Requisites:
    ✓ go is available on system
    ✓ ginkgo is available on system
    ✓ make is available on system
    ✓ git is available on system
    ✓ kubectl is available on system
    ✓ helm is available on system
✓ 'instaslice-operator' already exists. Skipping clone.
✓ Updated /home/csubrama/bharath/kruize-demos/monitoring/local_monitoring/gpu-demo/no-gpu/instaslice-operator/config/manager/manager.yaml successfully.
Running make test-e2e-kind-emulated ....................................................................................................... Done.
✗ make test-e2e-kind-emulated failed
```

Reply:
Can you please take a look at the local monitoring demo script, we already have common functions to do the install setup, clone repos etc.

Reply:
> Does this script setup kind? It failed for me at this step

Yes, it does.

 >  Can you please take a look at the local monitoring demo script, we already have common functions to do the install setup, clone repos etc.
 
 Yes I have tried them but the problem is instaslice uses kind as default kubernetes cluster to deploy and test emulated version of it. So I had to use existing instaslice setup and then install everything on top of it. 
 
 Can you please share me the content of `kruize_gpu_demo.log` to check the issue please?

Reply:
Here is the log file

[kruize_gpu_demo.log](https://github.com/user-attachments/files/18763535/kruize_gpu_demo.log)

Reply:
Thanks @chandrams 

```
Creating cluster "kind-e2e" ...
 • Ensuring node image (kindest/node:v1.31.1) 🖼  ...
 ✓ Ensuring node image (kindest/node:v1.31.1) 🖼
 • Preparing nodes 📦   ...
 ✗ Preparing nodes 📦 
Deleted nodes: ["kind-e2e-control-plane"]
ERROR: failed to create cluster: could not find a log line that matches "Reached target .*Multi-User System.*|detected cgroup v1"
make: *** [Makefile:226: create-kind-cluster] Error 1
```

I feel some issue with kind cluster start. I haven't seen this error at my end. 

Can you please check if you can start a kind cluster on your system?